### PR TITLE
Initial CRD status and e2e test

### PR DESCRIPTION
**Pull request by sjug**
So you can take a look at where we're at first thing Thursday @jmencak.

Status is set in CRD, tests pass on good status.

Let me know what you think.

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+func TestTunedStatus(t *testing.T) {
+	ctx, client, ns := prepareTest(t)
+	defer ctx.Cleanup()
+
+	t.Log("=== Wait for TunedDeployment to be ready")
+	if err := waitForTunedDeploymentReady(t, client, deploymentTimeout); err != nil {
+		t.Errorf("failed to wait for TunedDeployment to get ready: %s", err)
+	}
+	t.Logf("tunedDeplyoment in %s is ready", ns)
+
```
Comment (jmencak) [inline test/e2e/e2e_test.go]: Extra new line.

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	tuned "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1alpha1"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	deploymentTimeout = time.Minute * 5
+	apiTimeout        = 10 * time.Second
```
Comment (jmencak) [inline test/e2e/e2e_test.go]: Consistency, please.

**Comment by sjug** [comment]
@jmencak The `LastTransitionTime` should be set by the helper function `SetOperatorCondition`, I see it's null when I test it as well. I can set the field manually before I pass it into the function as a workaround? Thoughts?

**Comment by jmencak** [comment]
If `SetOperatorCondition` should set `LastTransitionTime`, I suggest filing an issue and not setting this manually.

**Comment by jmencak** [comment]
/lgtm

please create a relevant issue about the `LastTransitionTime` not being populated.

**Comment by sjug** [comment]
@jmencak `LastTransitionTime` fixed.

**Pull request merged**